docs/processes/project-lifecycle.md

```diff
@@ -22,7 +22,7 @@ Find the template: [Idea Draft](https://www.notion.so/Idea-Draft-1859a4c092c780a
 
 Once the Idea Draft is approved, the Partner Lead, Team Lead and Architect will move on to the Tech Design. This document provides an overview of each contract, external function, and off-chain component, defining how every piece of functionality works and outlining a time estimate. To mitigate risks, we also break the project into smaller milestones—each with its own timeline and deliverables—to keep everyone aligned and help achieve the partner’s goals efficiently.
 
-While the project is still being defined and refined, we start preparing the advanced tests, by collecting invariants. This initial collection is often a first draft, as many of these might still change during the development.
+While the project is still being defined and refined, we begin preparing advanced tests by collecting invariants. This initial collection typically serves as a first draft, since multiple invariants may change during development. Additionally, a member of our internal security team reviews the Tech Design early to identify potential architectural issues before we proceed too far into development. Neither of these processes need to block the start of development, but we aim to initiate them as soon as our on-chain security team has the necessary availability.
```

Comment:
```suggestion
While the project is still being defined and refined, we begin preparing advanced tests by collecting invariants. This initial collection typically serves as a first draft, since multiple invariants may change during development. Additionally, a member of our internal security team reviews the Tech Design early to identify potential architectural issues before we proceed too far into development. Neither of these processes needs to block the start of development, but we aim to initiate them as soon as our on-chain security team has the necessary availability.
```

Reply:
fixed on https://github.com/defi-wonderland/handbook/pull/17/commits/63fa1382562f02af731766186b0380300aab1803